dvc/command/config.py

```diff
@@ -98,7 +98,7 @@ def run(self):
 
 
 def add_parser(subparsers, parent_parser):
-    CONFIG_HELP = "Get or set config options."
+    CONFIG_HELP = "Get or set config settings."
```

Comment:
`config settings` seems like a tautology. `dvc config` is about managing config options, I don't think we should use the term "setting" here.

Reply:
OK, that makes sense. But the reason I suggest this change is that the update I'm sending for the `dvc config` ref. changes the terminology to differentiate between command options (`--unset`, `-v`, etc) from the DVC ~~options``settings that are managed via that command.

So maybe "Get or set DVC settings"? What do you think @shcheklein ? (Since you're reviewing that docs PR)

Reply:
@jorgeorpinel I don't have a strong preference on this to be honest. If git is using "config option" terminology like @efiop mentioned let's just keep that one and don't change it in the docs as well. What was your initial motivation/thought process, @jorgeorpinel, to change it?

Reply:
p.s. the PR is https://github.com/iterative/dvc.org/pull/331

Reply:
The motivation is explained in the previous comment, https://github.com/iterative/dvc/pull/1999#discussion_r283924254

But sure, lets keep "config options" then 😃  I'll close this PR and revert in my fork.